docs/configuration/magento2-testing.md

```diff
@@ -135,6 +135,19 @@ There's one thing you should be aware of: **always provide full path to `phpunit
 
 If you have [configured Xdebug](xdebug.md), run Integration tests inside **Debug** console (`warden debug` instead of `warden shell`). The code execution will stop at the breakpoints.
 
+
+### Troubleshooting
+
+- In case if you're getting message like `Fatal error: Allowed memory size of ...` try to add prefix `php -dmemory_limit=-1` to your command, like 
```

Comment:
```suggestion
- In case you're getting a message like `Fatal error: Allowed memory size of ...` try to add prefix `php -dmemory_limit=-1` to your command, like 
```